pkg/experiment/block/section_profiles.go

```diff
@@ -33,15 +33,13 @@ func openProfileTable(_ context.Context, s *Dataset) (err error) {
 			s.inMemoryBucket(buf), s.obj.path, offset, size,
 			0, // Do not prefetch the footer.
 			parquet.SkipBloomFilters(true),
-			parquet.SkipPageIndex(true),
```

Comment:
To be more error prone, we could also first try to open with page index and if that fails, fall back to skipping the index.